### ISSUE TITLE
Add functionality to OC admin UI to mount a series somewhere in Tobira

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#352]
Opencast administrators, when creating a new series in the admin-UI, should be able to add that series somewhere in Tobira. 

To offer this feature, we will add a new "Tobira" stop in the "Create series" wizard. That step will likely come after the "meta data" and "access policy" steps. In this step, the page structure of Tobira is shown in a way that can be navigated. 

- The user can add new pages in this dialogue (setting page title and path segment), also subpages of newly created pages. All of these new pages are not persisted in any database at all until the "create series" wizard is completely finished. 
- When selecting any empty page, the user can press a "mount series here" button. 
- The dialogue only allows mounting the series in one page. The selected mount-page is shown somewhere. If no mount-page has been selected yet, this is also shown somewhere, e.g. "this series is not added to the video portal". 
- The dialogue also just allows creating new empty pages that are super-pages of the selected "mount-page": unrelated pages cannot be created here. This somehow needs to be communicated to the user.
- When a mount point for the series is selected and the "create series" wizard finished, the new page (and any intermediate pages) are created in Tobira. The mount-page in Tobira will show all events from that series. The series/page will also be findable through Tobira's search.

**Explicitly not part of this issue:**
- Any authorization in the admin-UI: all users who can create a series can also use the Tobira dialogue. 
- Changing the mount point of existing series.
- Modifying any additional page settings apart from title, path-segment and the "show series here" selection. -> This should be done in Tobira

### Technical notes

- Soft-blocked by #351

### Open questions

- [ ] Where do we implement it? As fork? Contribute it to community immediately and make it disabled by default? ...?
- [ ] Is it required to be able to add a series to an existing realm/page? 
- [ ] How is this done? Synchronous? Who talks to Tobira: admin-UI frontend or OC backend? What if Tobira is not available at that moment?

**Comment by LukasKalbertodt** [comment]
Done in https://github.com/elan-ev/opencast-tobira/pull/72
